torch/fx/experimental/symbolic_shapes.py

```diff
@@ -2087,6 +2164,16 @@ def simplify(self, expr: "sympy.Expr") -> "sympy.Expr":
             # divisions simplified away
             if new_pows.issubset(pows) and new_rationals.issubset(rationals):
                 expr = new_expr
+        if isinstance(expr, sympy.Rel):
+            lhs_const = get_added_const(expr.lhs)
+            rhs_const = get_added_const(expr.rhs)
+            if (
+                    lhs_const is not None
+                    and rhs_const is not None
```

Comment:
Why does the `rhs_const` need to not be none here?

Reply:
Hmm. Good question. Since we want to possibly isolate a symbol on the left-hand side, maybe we could ignore what is in the right-hand side.